indexer/cell_coverer.hpp

```diff
@@ -98,3 +98,115 @@ inline void CoverRect(m2::RectD rect, size_t cellsCount, int maxDepth, vector<Ce
     result.push_back(id);
   }
 }
+
+template <typename BoundsT, typename CellIdT>
+void CoverRanked(m2::RectD rect, int maxDepth, vector<CellIdT> & result)
```

Comment:
Probably it's worth adding a comment to this method.

Reply:
done